Use nginx_buildpack instead of staticfile_buildpack (#7)
alphagov/paas-ip-safelist-service: README.md

> # Reliability Engineering IP Whitelisting Route Service

Reviewer: ```suggestion
# Reliability Engineering IP Safelisting Route Service
```

Author: Renamed to `GOV.UK PaaS IP authentication route service`

This repo contains a simple Nginx application which acts as a proxy for your
web applications and provides an IP restriction layer.

This repo is a template, which you should customise according to your needs
using the application manifest.

All PaaS traffic will go through the route service to filter traffic.

## Requirements

* Cloud Foundry CLI (https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)

You should log in using the Cloud Foundry CLI
(https://docs.cloud.service.gov.uk/#setting-up-the-command-line).

> For all actions you should always make sure you selected the space you intend

Reviewer: ```suggestion
For all actions you should always make sure you are targeting the correct space.
```

Author: Done in `e05ca23`

> to target.

Reviewer: ```suggestion
```
